moztelemetry/scalar.py

```diff
@@ -110,4 +110,7 @@ def _get_scalar_definition(url, metric):
         else:
             definitions = Scalar._definition_cache[url]
 
+        if metric.lower() not in definitions:
+            raise KeyError("Definition not found for {}".format(metric))
```

Comment:
What about using a custom exception? That way calling code can except this specific error and not a generic KeyError.